tests/unit/models/cart-model-test.js

```diff
@@ -96,6 +96,29 @@ test('it creates a line item when you add a variant', function (assert) {
   });
 });
 
+test('it creates a line item when you add a variant', function (assert) {
+  assert.expect(3);
+
+  const done = assert.async();
+
+  const quantity = 2;
+
+  const variant = singleProductFixture.product_listing.variants[1];
+
```

Comment:
@minasmart what I did instead was to get the initial quantity if a line item exists. Then add that to the quantity.

It just didn't feel right to test if it exists first.